### PR TITLE
Ls 1

### PR DESCRIPTION
**Pull request by IgorOksentyuk**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,603 @@
...
+    <main>
+      <section class="education-block">
+        <div class="education-block____info">
+          <h1>Education is the most powerfull weapon</h1>
+          <p>
+            A bank is a financial institution accepts deposits from public and creates credit. For
+            all of your needs.
+          </p>
+          <button>Read More</button>
+          <a href="#form"><button>Get started</button></a>
```
Comment (alexsednev) [inline index.html]: The element "button" must not appear as a descendant of the "a" element.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,603 @@
...
+          <div>
+            <p>
+              Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque
+            </p>
+            <p>
+              Laudantium, totam rem aperiam, eaque ipsa quae ab illo inventore veritatis et quasi
+              architecto beatae vitae dicta sunt explicabo.
+            </p>
+          </div>
+          <a href="#form"><button>Apply Now</button></a>
```
Comment (alexsednev) [inline index.html]: The element "button" must not appear as a descendant of the "a" element.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,603 @@
...
+            <svg width="9" height="14" fill="none">
+              <use xlink:href="images/sprite.svg#arrow-left"></use>
+            </svg>
+          </div>
+          <div class="courses-block___item">
+            <img src="/images/courses-block/man_1.png" alt="man_1" />
+            <img width="270" height="364" src="/images/courses-block/desing.png" alt="desing" />
+            <p>DESIGN</p>
+            <p>Learn Photoshop title</p>
+            <span>____</span>
```
Comment (alexsednev) [inline index.html]: In this place should be a horizontal line. Span with underline don't fits responsive layout.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,603 @@
...
+
+            <div>
+              <div class="tour-block____right-circle"></div>
+              <p>Neque porro quisquam est, qui dolorem ipsum quia dolor sit amet, consectetur</p>
+              <div class="tour-block____description-circle"></div>
+              <p>
+                Ut enim ad minima veniam, quis nostrum exercitationem ullam corporis suscipit
+                laboriosam, nisi ut aliquid ex ea commodi consequatur?
+              </p>
+            </div>
```
Comment (alexsednev) [inline index.html]: Here should be an ul list

File: index.html [hunk 5]
```diff
@@ -0,0 +1,603 @@
...
+              </svg>
+              <p>United States</p>
+            </div>
+          </div>
+        </div>
+
+        <div class="business-block____right">
+          <p>FEATURED</p>
+          <div class="business-block____right-remaining">
+            <p>$213 Days</p>
```
Comment (alexsednev) [inline index.html]: It's better to separate content in element cause another styles for $213. Same for next two elements.

File: index.html [hunk 6]
```diff
@@ -0,0 +1,603 @@
...
+            </p>
+            <p>Stephanie Magion</p>
+          </div>
+
+          <div class="customers-block____info-circles">
+            <img src="/images/customers-block/circle-blue.png" alt="circle-blue" />
+            <img src="/images/customers-block/circle-blue.png" alt="circle-blue" />
+            <img src="/images/customers-block/circle-purple.png" alt="circle-purple" />
+            <img src="/images/customers-block/circle-blue.png" alt="circle-blue" />
+            <img src="/images/customers-block/circle-blue.png" alt="circle-blue" />
```
Comment (alexsednev) [inline index.html]: Please don't use image for circle. You can create your own using html and css. For non-active dots you can set an lower oppacity as an example. Or another color. Don't use images for elements that you can create using html+css.

File: index.html [hunk 7]
```diff
@@ -0,0 +1,603 @@
...
+      </section>
+
+      <form action="#" id="form">
+        <input type="text" placeholder="Name Surname" />
+        <input type="number" placeholder="Phone" />
+        <input type="email" placeholder="Enter your email..." />
+        <button type="submit">Send</button>
+      </form>
+
+      <section class="sources-block">
```
Comment (alexsednev) [inline index.html:549]: Content of this section it's for footer.

File: index.html [hunk 8]
```diff
@@ -0,0 +1,603 @@
...
+        <svg width="53.12px" height="24px">
+          <use xlink:href="images/sprite.svg#union-icon"></use>
+        </svg>
+        <div class="sources-block-item">
+          <h3>Company</h3>
+          <p>About Company</p>
+          <p>Feature Course</p>
+          <p>Careers</p>
```
Comment (alexsednev) [inline index.html]: Please use ul list for this elements and next two as well.

File: index.html [hunk 9]
```diff
@@ -0,0 +1,603 @@
...
+
+            <span>______</span>
+          </div>
+        </div>
+      </section>
+
+      <section class="blog-block">
+        <div class="blog-block____title">
+          <h2>Latest Blog</h2>
+          <span>____</span>
```
Comment (alexsednev) [inline index.html]: You use this element more than one time. It's better create pseudo "after" and reuse it.

**Review by alexsednev — CHANGES_REQUESTED** [review]

File: index.html (hunk 9 above)
Comment (IgorOksentyuk) [reply index.html]: Ти про <span>____</span> чи про title взагалі?

File: index.html (hunk 8 above)
Comment (IgorOksentyuk) [reply index.html]: Done.

File: index.html (hunk 7 above)
Comment (IgorOksentyuk) [reply index.html:549]: Done.

File: index.html (hunk 6 above)
Comment (IgorOksentyuk) [reply index.html]: Done.

File: index.html (hunk 5 above)
Comment (IgorOksentyuk) [reply index.html]: Done.

File: index.html (hunk 4 above)
Comment (IgorOksentyuk) [reply index.html]: Done.

File: index.html (hunk 3 above)
Comment (IgorOksentyuk) [reply index.html]: Змінив span з нижнім підкресленням просто на span, згодом стилізую за допомогою css. Вірно?

File: index.html (hunk 9 above)
Comment (alexsednev) [reply index.html]: It's about line.

Comment (IgorOksentyuk) [reply index.html]: Got it!

Comment (alexsednev) [reply index.html]: You can create pseudo after.
For example. 
.underline::after {
  ...
}
And use it with H2 heading or whatever you need it

Comment (IgorOksentyuk) [reply index.html]: I understand, thank you!

File: index.html (hunk 1 above)
Comment (IgorOksentyuk) [reply index.html]: Done.

File: index.html (hunk 3 above)
Comment (alexsednev) [reply index.html]: Ok. And please don't use your native language in github topics.

File: index.html (hunk 2 above)
Comment (IgorOksentyuk) [reply index.html]: Done.

File: index.html (hunk 3 above)
Comment (IgorOksentyuk) [reply index.html]: Yes sir!)

**Review by alexsednev — APPROVED** [review]

**Pull request merged**